### ISSUE TITLE
Sunrise schedule executed twice

### ISSUE THREAD

**Issue by arthurdent75** [arthurdent75/SimpleScheduler#87]
I use it for all my scheduled automations and it is working fine, except for this entry:

![image](https://user-images.githubusercontent.com/60071192/194535813-26452717-d753-470b-8412-ef8a256463a7.png)

It is always executed twice every day with one or two minutes in between:

```
[Wed, 05 Oct 2022 07:45:01 +0200] SCHED:  script/turn_on {"entity_id":"script.lfx_scheduler_overdag_1"} 
[Wed, 05 Oct 2022 07:47:00 +0200] SCHED:  script/turn_on {"entity_id":"script.lfx_scheduler_overdag_1"}
```

It must be because of the sunrise trigger, because in the weekend it is just triggered only once at 9 o’clock.
I have other entries that use trigger sunset and they work fine.

**Comment by arthurdent75** [comment]
Find out why this is happening.
Will be solved in the next release!

**Comment by TonECCT** [comment]
That is great news!

**Comment by arthurdent75** [comment]
Solved in 2.0
